src/DynamicData/Internal/Defaults.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) 2011-2023 Roland Pheasant. All rights reserved.
+// Roland Pheasant licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for full license information.
+
+using System.Reactive.Concurrency;
+
+namespace DynamicData.Internal;
+
+internal static class Defaults
```

Comment:
@RolandPheasant Do you think this should be a public thing so that consumers can change the value if they want to?  We could call it `DynamicData.GlobalConfig` with this as a property or something.

It does sound like a foot-gun though.  Maybe it should just be some enum that allows us to ensure it only gets set to schedulers that will work.  Then again, maybe they have a reason for doing everything on the Intermediate Scheduler...  Who can say?

Reply:
I suspect in this case, making it internal would probably be sufficient and it allows up to change our mind and give the consumers the best choice out of the box.

Also when I added the binding options I accidentally added the global options class in the binding namespace. That should be moved to the root namespace. It would technically be a breaking change, but I'd suggest we should not increase the bug version as it would be correcting a mistake for a bit of code for which there's probably only one or two users.

Reply:
See DynamicDataOptions here https://github.com/reactivemarbles/DynamicData/blob/main/src/DynamicData/Binding/BindingOptions.cs

Reply:
Funny you should ask... I just moved it to the main namespace.

Are you asking me to move the BindingOptions as part of this PR?  They can't go in the same class if one is public and the other is internal.

Reply:
I moved it to the main namespace and gave it a better name but kept it as internal.  If you want, I can move the `DynamicDataOptions` as well (but it might be better as another PR).

Then later we can discuss adding a configurable Default Scheduler value to it...

Reply:
@RolandPheasant See #864 for those changes.